tools/rosnode/src/rosnode/__init__.py

```diff
@@ -554,7 +554,7 @@ def get_node_connection_info_description(node_api, master):
         raise ROSNodeIOException("Communication with node[%s] failed!"%(node_api))
     return buff
 
-def rosnode_info(node_name):
+def rosnode_info(node_name, non_verbose):
```

Comment:
Please avoid inverted logic. Either call the argument `quiet` or `verbose`.

Adding a positional argument breaks existing code. Please add the new argument as an optional keyword argument. And the default value should not change the current behavior.